antora-ui-camel/src/css/blog.css

```diff
@@ -119,6 +119,24 @@ article.blog p {
   background: var(--color-smoke-90);
 }
 
+.avatar {
+  vertical-align: middle;
+  width: 100px;
+  height: 100px;
+  border-radius: 50%;
+}
+
+#publishDate {
+  position: relative;
+  bottom: 35px;
+}
+
+.date-and-name {
+  position: relative;
+  bottom: 105px;
+  left: 110px;
+}
+
```

Comment:
- These rules need to be scoped to `.blog`, as all other rules in this file are.
- Don't use `px`, we exclusively use relative lengths
- Don't use id-based rules for elements that could occur more than once

Reply:
Replaced `px` with `em`.
Removed id-based rules and scoped the rules to .blog